src/CollapsibleNav.js

```diff
@@ -19,41 +18,48 @@ const CollapsibleNav = React.createClass({
     eventKey: React.PropTypes.any
   },
 
-  getCollapsibleDOMNode() {
-    return React.findDOMNode(this);
-  },
 
-  getCollapsibleDimensionValue() {
-    let height = 0;
-    let nodes = this.refs;
-    for (let key in nodes) {
-      if (nodes.hasOwnProperty(key)) {
+  // getCollapsibleDimensionValue() {
```

Comment:
I was hoping someone might know why this was used instead of the normal default method of getting height, it tested it without and it seems to work fine